clients/flash/air-client/src/org/bigbluebutton/air/main/views/MainView.as

```diff
@@ -35,6 +37,14 @@ package org.bigbluebutton.air.main.views {
 			addElement(_presentationView);
 			
 			// add deskshare view here and position like the presentation view
+			_screenshareView = new ScreenshareDock();
+			_screenshareView.percentWidth = 100;
+			_screenshareView.percentHeight = 100;
+			
+			// Listen for using stage view as we have to hide some views to make stage view visible.
+			_screenshareView.addScreenshareRunningListener(onScreenshareRunning);

```

Comment:
I think it would be best to create a dispose() function in MainView and have it remove the listener to clean up any traces.